airbyte-cdk/python/airbyte_cdk/models/airbyte_protocol.py

```diff
@@ -2,4 +2,59 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 #
 
-from airbyte_protocol.models import *
+from dataclasses import InitVar, dataclass
+from typing import Annotated, Any, Dict, List, Mapping, Optional
+
+from airbyte_protocol_dataclasses.models import *
+from serpyco_rs.metadata import Alias
+
+
+@dataclass
+class AirbyteStateBlob:
+    kwargs: InitVar[Mapping[str, Any]]
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        # Set any attribute passed in through kwargs
+        for arg in args:
+            self.__dict__.update(arg)
+        for key, value in kwargs.items():
+            setattr(self, key, value)
+
+    def __eq__(self, other: object) -> bool:
+        return False if not isinstance(other, AirbyteStateBlob) else bool(self.__dict__ == other.__dict__)
+
+
+# The following dataclasses have been redeclared to include the new version of AirbyteStateBlob
```

Comment:
why does AirbyteStateBlob needs an updated definition? I feel like that should be explained in the code

Reply:
added docstring